src/main/java/frc/robot/Constants.java

```diff
@@ -88,9 +88,30 @@
     // amount of time to shoot the second ball
     public static final double SHOOT_BALL2_WAIT_TIME = 0.75;
     
-    public static final double TARMAC_DEFAULT_DISTANCE = 94.0;//12.0 * 7.0;  // 9ft minus hub radius
+    //
+    // Various constants used for shooting
+    //
 
-    // chute subsystem
+    // The shooting mechanism is about 24" from the edge of the robot, including bumper
+    // We therefore subtract SHOOTER_POS from all shooting distance constants
+    public static final double SHOOTER_POS = 24;
+    
+    // Our usual "Tarmac" shot is 9ft
+    public static final double TARMAC_DEFAULT_DISTANCE = (12.0 * 9.0) - SHOOTER_POS;
```

Comment:
This was tweaked at Revere to be a bit longer, 94 in. Should be set back to that value.